java/hsfs/src/main/java/com/logicalclocks/hsfs/metadata/HopsworksInternalClient.java

```diff
@@ -109,6 +113,9 @@ private Registry<ConnectionSocketFactory> createConnectionFactory()
     Properties systemProperties = System.getProperties();
 
     Path trustStorePath = Paths.get(systemProperties.getProperty(DOMAIN_CA_TRUSTSTORE));
+    if (trustStorePath == null || !Files.exists(trustStorePath)) {
+      trustStorePath = Paths.get(T_CERTIFICATE);
```

Comment:
Shouldn't this use the `MATERIAL_DIRECTORY` as well?

Reply:
It is already initialized in the constants above

Reply:
Ok true.